### ISSUE TITLE
Update datafile decoding for alternative datasets

### ISSUE THREAD

**Issue by shaunwbell** [NOAA-Strausz/EcoFOCI_ssmi_ice#1]
The NASA bootstrap data has a different format than the NRT data - the code below seems to work and is based on the discussion at -[https://nsidc.org/data/nsidc-0079/versions/3#version]( https://nsidc.org/data/nsidc-0079/versions/3#version)

```python
def decode_datafile(filename,datasource='NRT'):
    icefile = open(filename, 'rb')

    if datasource is 'NRT':
       #remove the header
       icefile.seek(300)
       ice = np.fromfile(icefile,dtype=np.uint8)
       ice[ice >= 253] = 0
       ice = ice/2.5
    elif datasource is 'Bootstrap':
       #no header
       ice = np.fromfile(icefile,dtype=np.uint16)
       ice = ice/10.
       ice[ice == 110] = 0 #110 is land
       ice[ice == 120] = 100 #120 is polar hole
    else:
       ice=np.nan

    return ice;
```

**Comment by shaunwbell** [comment]
the definition of land and polar hole looks erroneous in the description (at least in 1978-1983)

make all land 0 and polar hole full (100)
```python
       ice[ice == 110] = 100 #110 is polar hole
       ice[ice == 120] = 0 #120 is land
```

**Comment by shaunwbell** [comment]
Closed by #2
